src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/EditAndSyncFeatures/EditAndSyncFeatures.xaml.cs

```diff
@@ -0,0 +1,511 @@
+
+            // Start the job
+            _generateGdbJob.Start();
+        }
+
+        private async void HandleGenerationStatusChange(GenerateGeodatabaseJob job)
+        {
+            JobStatus status = job.Status;
+
+            // If the job completed successfully, add the geodatabase data to the map
+            if (status == JobStatus.Succeeded)
+            {
+                // Clear out the existing layers
+                MyMapView.Map.OperationalLayers.Clear();
+
+                // Get the new geodatabase
+                _resultGdb = await job.GetResultAsync();
+
+                // Loop through all feature tables in the geodatabase and add a new layer to the map
+                foreach (GeodatabaseFeatureTable table in _resultGdb.GeodatabaseFeatureTables)
+                {
+                    // Create a new feature layer for the table
+                    FeatureLayer _layer = new FeatureLayer(table);
+
+                    // Add the new layer to the map
+                    MyMapView.Map.OperationalLayers.Add(_layer);
+                }
+
+                // Enable editing features
+                _readyForEdits = EditState.Ready;
+            }
+
+            // See if the job failed
+            if (status == JobStatus.Failed)
+            {
+                // Create a message to show the user
+                string message = "Generate geodatabase job failed";
+
+                // Show an error message (if there is one)
+                if (job.Error != null)
+                {
+                    message += ": " + job.Error.Message;
+                }
+                else
+                {
+                    // If no error, show messages from the job
+                    var m = from msg in job.Messages select msg.Message;
+                    message += ": " + string.Join<string>("\n", m);
+                }
+
+                // Show the message
+                ShowStatusMessage(message);
+            }
+        }
+
+        private void HandleSyncStatusChange(SyncGeodatabaseJob job)
+        {
+            JobStatus status = job.Status;
+
+            // Tell the user about job completion
+            if (status == JobStatus.Succeeded)
+            {
+                ShowStatusMessage("Sync task completed");
+            }
+
+            // See if the job failed
+            if (status == JobStatus.Failed)
+            {
+                // Create a message to show the user
+                string message = "Sync geodatabase job failed";
+
+                // Show an error message (if there is one)
+                if (job.Error != null)
+                {
+                    message += ": " + job.Error.Message;
+                }
+                else
+                {
+                    // If no error, show messages from the job
+                    var m = from msg in job.Messages select msg.Message;
+                    message += ": " + string.Join<string>("\n", m);
+                }
```

Comment:
Could you expand the above 2 lines of code into a few more lines? Getting information from the "job" variable (GenerateGeodatabasJob) is actually a big deal for users of the ArcGIS Runtime to know about and is probably not talked about much in other samples of the Library Reference Documentation. Expounding these lines of code into individual variables will give developers deeper insight into what can be done with the results of a Geoprocessing task. Again coding "run-on sentences" are nice for shortened reading but do not explain much of what is actually occurring.